content/information-distribution/source.md

```diff
@@ -23,12 +58,221 @@ one key vs all of the keys vs the source's entire Matrix account), and take the
 actions it deems necessary.
 
 In such an event, client implementations **should** consider articles posted
-prior to the key being compromised as possibly, but not surely trustworthy.
+prior to the key being declared as compromised as possibly, but not surely
+trustworthy.
+
+## Localisation
+
+An information website might want to publish articles in more than one language.
+In such case, it **can** create sub-sources, each of which handling the
+publication of its articles in one language. A source, regardless of whether it
+is a sub-source or not, **must not** publish articles in more than one language.
+
+A sub-source is an entity similar to a source, with the exception that it
+**must** be related to a source. Each sub-source **must** be a registered Matrix
+user, and **must** register itself on the Matrix room through the publication of
+a state event of the `network.informo.subsource` type, with the Matrix user's ID
+as the event's state key. The event's content **must** be embedded in a [signed
+Matrix event](/information-distribution/signature/#signed-matrix-event), signed
+by one of the parent source's public keys, with its `signed` object using the
+following model:
+
+|      Parameter      |    Type    | Req. |                                  Description                                   |
+| ------------------- | ---------- | :--: | ------------------------------------------------------------------------------ |
+| `sig_algo`          | `string`   |  x   | Algorithm the sub-source will use to cryptographically sign its articles. 🔧    |
+| `sig_keys`          | `[string]` |  x   | Public keys the sub-source will use to cryptographically sign its articles. 🔧  |
+| `website`           | `string`   |      | URL of the source's website in this language, if there's one.                  |
+| `description`       | `string`   |      | Short localised description of the source and its publications.                |
+| `custom`            | `object`   |      | Additional information for custom client implementations.                      |
```

Comment:
I'd assume that case wouldn't happen very often, since in most cases you'd look for a localised version of a source (so going from the source to the sub-source) rather than for what source does a localised version belong.

I'd picture client implementations only going either "here are all of the localisations of that source" or "here are all of the sources that offer a localisation for that language", both of them being doable by processing only `network.informo.source` events.

IMHO, sub-sources would really only exist as a way to retrieve articles in a specific language rather than anything else.

Reply:
Gotcha. I'll add it just in case.